frontend/integration-tests/tests/kubevirt/mocks.ts

```diff
@@ -289,6 +289,33 @@ export const cloudInitCustomScriptConfig = {
   customScript: basicVmConfig.cloudInitScript,
 };
 
+export const examplePod = (options) => {
```

Comment:
it is a bit easier to read if you use object deconstruction here

Reply:
done